back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -68,7 +68,7 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     if: proc { |object, _params| object.supports_linear_scale_labels? }
 
   attributes :select_count_enabled, :maximum_select_count, :minimum_select_count, if: proc { |object, _params|
-    object.multiselect?
+    object.supports_select_count?
```

Comment:
This works fine like this, but in some BE CDM we decided that we wouldn't use `if: proc` anymore unless it's really necessary (e.g. the information is sensitive and should remain secret under certain conditions)

Reply:
Is there a preferred pattern instead? I changed it but not sure if this is the best way to go.